app/src/main/java/nerd/tuxmobil/fahrplan/congress/system/OnBootReceiver.java

```diff
@@ -21,6 +17,9 @@
 import nerd.tuxmobil.fahrplan.congress.models.SchedulableAlarm;
 import nerd.tuxmobil.fahrplan.congress.repositories.AppRepository;
 import nerd.tuxmobil.fahrplan.congress.utils.FahrplanMisc;
+import org.ligi.tracedroid.logging.Log;
+
+import java.util.List;
```

Comment:
# Organizing imports
Android Studio 3.6 does not rearrange the original imports for me. Are you using a custom setting?

Reply:
I did not change any settings before this PR. I only recently set "class to count to use import with *" to 999 to avoid unwanted .* imports.

Reply:
Can you please move the part of the "undo" commit 77f3eef9c574334186538c4c7292284f9165348f into the original commit 7d28aec26cbaef326baa136546029d47531d3250 where the imports were reorganized to clean up the history?

Reply:
Resolved in the "Moment" rewrite of the branch. See 29c7fd82e7a7bf2716f64ab1b1afed2f1eca7233.